opent2t.p.hub/opent2t.p.hub.raml

```diff
@@ -0,0 +1,57 @@
+#%RAML 0.8
+title: opent2t.p.hub
+version: v1.0
+
+schemas:
+  - Platform: !include opent2t.p.hub.json
+
+traits:
+  - interface-sensor:
+      queryParameters:
+        if:
+          enum: ["oic.if.s", "oic.if.baseline"]
+
+/platforms:
```

Comment:
#Resolved Added default get and get(expand)

---
In reply to: [88085084](https://github.com/openT2T/translators/pull/117#discussion_r88085084) [](ancestors = 88085084)